fediplay/queue.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, cache_dir):
         self.cache_dir = cache_dir
 
     def _progress_hook(self, progress):
-        if progress['status'] == 'downloading' and progress['filename'] not in self.filenames:
+        if progress['status'] == 'finished' and progress['filename'] not in self.filenames:
```

Comment:
Does this still work with albums? We originally changed this to `'downloading'` to make those work.

Maybe we need to wait until we receive the final `'finished'` (IIRC, there was just one?) before beginning playback.

Reply:
It seems to work. I've tested it with both bandcamp albums and a youtube playlist.

Not sure why `finished` outputted just one before and not anymore, changes in youtube-dl perhaps? Hmm.

Reply:
Hm, well, you made the original change, so we can roll with it 🙂 

Out of curiosity, re #21, what OS are you using?

Reply:
:D

OS was Debian (stretch)